test/functional/apps/discover/_discover.js

```diff
@@ -0,0 +1,134 @@
+define(function (require) {
+  var Common = require('../../../support/pages/Common');
+  var SettingsPage = require('../../../support/pages/SettingsPage');
+  var DiscoverPage = require('../../../support/pages/DiscoverPage');
+  var expect = require('intern/dojo/node!expect.js');
+
+  return function (bdd, scenarioManager) {
+    bdd.describe('discover app', function describeIndexTests() {
+      var common;
+      var settingsPage;
+      var discoverPage;
+      var remote;
+      var fromTime;
+      var toTime;
+      this.timeout = 3 * 60 * 1000;
```

Comment:
It's not clear to me why the discover app tests failed before the first test ran in this instance
https://travis-ci.org/elastic/kibana/builds/90989770
"Error: Request Timeout after 30000ms"
While in this run they all passed in about 22 seconds.  I think the bdd.before must have been taking more than 8 seconds, so that time plus the test time exceeded 30 seconds.  I have a 60 second timeout in the last test but I think the timeout needs to be set before the bdd.before (and I probably don't need as long, and don't need the timeout in the bdd.it test).
https://travis-ci.org/elastic/kibana/builds/91003284
PASS: firefox on any platform - discover app - discover app - query - should show correct time range string (32ms)

>> PASS: firefox on any platform - discover app - discover app - query - save query should show toast message and display query name (6791ms)

>> PASS: firefox on any platform - discover app - discover app - query - load query should show query name (4179ms)

>> PASS: firefox on any platform - discover app - discover app - query - should show the correct bar chart (11402ms)